### PR TITLE
Adnuntius: Max deals

### PR DESCRIPTION
**Pull request by mikael-lundin**
This will allow Prebid server bids to pass a parameter called "maxDeals" that sets the adapter up for passing multiple bids back to prebid client. 

I did get an error in the:
/workspaces/prebid-server/config/config_test.go:495

But since I hadn't changed anything I can't take credit for that error in the test. :) 

The documentation change is adressed in this PR:
https://github.com/prebid/prebid.github.io/pull/4711

### CODE REVIEW

File: adapters/adnuntius/adnuntius.go [hunk 1]
```diff
@@ -344,48 +392,35 @@ func generateBidResponse(adnResponse *AdnResponse, request *openrtb2.BidRequest)
...
-				ADomain: adDomain,
-			}
-
 			bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
-				Bid:     &bid,
+				Bid:     adBid,
 				BidType: "banner",
 			})
+
+			fmt.Println(adunit.Deals)
```
Comment (onkarvhanumante) [inline adapters/adnuntius/adnuntius.go:409]: seems like debug leftover. fmt line should be removed

File: openrtb_ext/imp_adnuntius.go [hunk 2]
```diff
@@ -4,4 +4,5 @@ type ImpExtAdnunitus struct {
 	Auid      string `json:"auId"`
 	Network   string `json:"network"`
 	NoCookies bool   `json:noCookies`
+	MaxDeals int   `json:maxDeals`
```
Comment (onkarvhanumante) [inline openrtb_ext/imp_adnuntius.go]: couple of changes needed here:

- missing double quotes in json tag for `noCookies` and `maxDeals`
- should fix space to align `MaxDeals` with other members of `ImpExtAdnunitus`

```
type ImpExtAdnunitus struct {
	Auid      string `json:"auId"`
	Network   string `json:"network"`
	NoCookies bool   `json:"noCookies"`
	MaxDeals  int    `json:"maxDeals"`
}
```

```
   Auid            string    `json:"auId"`
   Network     string    `json:"network"`
   NoCookies bool      `json:noCookies`
   
```

Comment (onkarvhanumante) [inline openrtb_ext/imp_adnuntius.go]: should update bidder parms json and add docs PR for the new `maxDeals`  param

File: adapters/adnuntius/adnuntius.go [hunk 3]
```diff
@@ -27,31 +27,35 @@ type adnAdunit struct {
 	AuId       string    `json:"auId"`
 	TargetId   string    `json:"targetId"`
 	Dimensions [][]int64 `json:"dimensions,omitempty"`
+	MaxDeals int `json:"maxDeals,omitempty"`
```
Comment (onkarvhanumante) [inline adapters/adnuntius/adnuntius.go]: should fix space to align `MaxDeals` with other members of `adnAdunit`

File: openrtb_ext/imp_adnuntius.go [hunk 4]
```diff
@@ -4,4 +4,5 @@ type ImpExtAdnunitus struct {
 	Auid      string `json:"auId"`
 	Network   string `json:"network"`
 	NoCookies bool   `json:noCookies`
+	MaxDeals  int    `json:maxDeals`
```
Comment (onkarvhanumante) [inline openrtb_ext/imp_adnuntius.go:7]: couple of changes needed here:

- missing double quotes in json tag for `noCookies` and `maxDeals`
- should fix space to align `MaxDeals` with other members of `ImpExtAdnunitus`

```
type ImpExtAdnunitus struct {
	Auid      string `json:"auId"`
	Network   string `json:"network"`
	NoCookies bool   `json:"noCookies"`
	MaxDeals  int    `json:"maxDeals"`
}
```

**Comment by Sonali-More-Xandr** [comment]
@mikael-lundin Please take a look at the comments above

**Comment by mikael-lundin** [comment]
Hi, sorry, had a bit of a vacation, my latest push should address the things above.

**Review by onkarvhanumante — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**